.gitignore

```diff
@@ -73,21 +73,21 @@ dist
 build
 
 # vscode config
-**/.vscode/settings.json
+.vscode/settings.json
```

Comment:
Removed `**/` for compatibility with `@local/esconfig/generate-ignore-patterns.cjs`. Adding `/` makes thig glob workspace-specific (which we don't want).